Cartfile

```diff
@@ -13,3 +13,4 @@ github "path/FastImageCache"		    "1.5.1"
 github "facebook/pop"			    "1.0.10"
 github "xmartlabs/Eureka" 		    "4.2.0"
 github "venmo/Static"			    "v2.5.0"
+github "Dashlane/SwiftDomainParser" "feature/perfomanceIsAFeature"
```

Comment:
nice, this way seems much better. this branch doesn't seem to be kept as up to date as master though. maybe ask them if they want to merge the branch into master? 

alternatively we could fork it

Reply:
is it possible that this is already provided by upstream? @fmarier mentioned that iOS Firefox should have this functionality

Reply:
Bear in mind that I'm totally unfamiliar with this code base, but this is the kind of facility I was expecting to see: https://github.com/brave/brave-ios/blob/0158c51c503a151facbab2b8b0d01950b6096050/SharedTests/NSURLExtensionsTests.swift#L81-L85

Reply:
@fmarier wow! thanks for helping out! This code was unknown to me.
@diracdeltas I will revert the lib and make changes using this code. Will also update the public suffix dat file.